registration/tests/serializers/test_family.py

```diff
@@ -1,19 +1,24 @@
 from django.test.testcases import TestCase
+from datetime import datetime
 
 from registration.models import Family, Student
+from enrolments.models import Enrolment, Session, Class
+from accounts.models import User
 from registration.serializers import FamilySerializer, StudentSerializer
+import pytz
 
 
 class FamilySerializerTestCase(TestCase):
     def setUp(self):
         self.parent1 = Student.objects.create(
             first_name="Merlin", last_name="Fish", role=Student.PARENT
         )
-
         self.parent2 = Student.objects.create(
             first_name="Gandalf", last_name="Whale", role=Student.PARENT
         )
-
+        self.parent3 = Student.objects.create(
```

Comment:
since this test file is getting kinda big, it would be nice to move creating all test objects specifically for `test_family_status` under that test for readability